src/styles/index.scss

```diff
@@ -4,0 +1,4 @@
+@import '../utils/variables';
+@import '../blocks/page';
```

Comment:
Ensure that the file '../blocks/page.scss' exists and is correctly referenced. If it doesn't exist, it will cause a compilation error.